utils/helm/speckle-server/templates/server/deployment.yml

```diff
@@ -158,6 +158,21 @@ spec:
             value: {{ .Values.server.migration.movedTo }}
             {{- end }}
 
+          # *** Gendo render module ***
+          - name: FF_GENDOAI_MODULE_ENABLED
+            value: {{ .Values.featureFlags.gendoModuleEnabled }}
+            
+            {{- if .Values.featureFlags.gendoModuleEnabled }}
+          - name: GENDOAI_KEY
+            value: {{ .Values.server.gendoKey }}
```

Comment:
The two keys need to be secret references, they should be loaded from the `server-vars` secret